data/json/vehicle_parts.json

```diff
@@ -4880,6 +4880,25 @@
         ]
     },{
         "type" : "vehicle_part",
+        "id" : "chimes",
+        "name" : "chimes",
+        "symbol" : "&",
+        "color" : "red",
+        "broken_symbol" : "&",
+        "broken_color" : "light_gray",
+        "damage_modifier" : 0,
+        "durability" : 40,
+        "epower" : -200,
+        "item" : "chimes",
+        "difficulty" : 4,
```

Comment:
Please add ```"location" : "on_roof",``` after that line (like it done in the solar panel definition), it will put the chimes on the roof (Sorry for late answer, I had some technical issues with my PC).